src/public/style.css

```diff
@@ -1,12 +1,72 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather|Open+Sans');
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Open Sans', sans-serif;
+}
+
 body {
-  background-color: #d9e6fc;
+  font-family: 'Merriweather', serif;
+}
+
+
+
+header {
+  margin: 0 !important;
+  display: block;
+}
+
+div.page {
+    background-color: lightgrey;
+    }
+
+div.title {
+  position: static;
+}
+
+div.content-container {
```

Comment:
You can target just the class name, without specifying `div`. This works too, but these styles wouldn't be applied to any other element type with `content-container` class.